### ISSUE TITLE
Where is the extension configuration page? Link needed in the wiki, possibly also in the readme?

### ISSUE THREAD

**Issue by Zazcallabah** [sjkp/letsencrypt-siteextension#318]
The installation page shows screenshots from the configuration page. It would be nice if it could also link to it (https://<sitename>.scm.azurewebsites.net/letsencrypt/).

**Comment by bambuca** [comment]
1. Navigate to you App Service Extensions:
1. Select your extension (Azure Let's Encrypt or Azure Let's Encrypt (No Web Jobs))
1. Click Browse
![image](https://user-images.githubusercontent.com/4871907/61419090-ffe60380-a8fc-11e9-8d39-647fb39c5640.png)

**Comment by Zazcallabah** [comment]
Yes exactly, clicking that link takes you to the url i posted above. It took me quite a while to find it. So maybe this image and those steps could be added to the wiki?

**Comment by modemgeek** [comment]
you used to be able to go to the hostname page directly by using the .scm.azurewebsites.net/letsencrypt/Home/install link.  However, if it is the first time viewing the page, it now give this error.  has anyone noticed or getting this error as well?


Line 22:             @Html.LabelFor(model => model.Hostnames, htmlAttributes: new { @class = "control-label col-md-2" }) Line 23:             <div class="col-md-10"> Line 24:                 @Html.ListBoxFor(model => model.Hostnames, ViewBag.HostNames as IEnumerable<SelectListItem>, new { htmlAttributes = new { @class = "form-control" } }) Line 25:                 @Html.ValidationMessageFor(model => model.Hostnames, "", new { @class = "text-danger" }) Line 26:             </div>
--



Line 22:             @Html.LabelFor(model => model.Hostnames, htmlAttributes: new { @class = "control-label col-md-2" }) Line 23:             <div class="col-md-10"> Line 24:                 @Html.ListBoxFor(model => model.Hostnames, ViewBag.HostNames as IEnumerable<SelectListItem>, new { htmlAttributes = new { @class = "form-control" } }) Line 25:                 @Html.ValidationMessageFor(model => model.Hostnames, "", new { @class = "text-danger" }) Line 26:             </div>
--

**Comment by modemgeek** [comment]
You used to be able to .scm.azurewebsites.net/letsencrypt/Home/install directly so you didn't have to go through the configuration page.  Now it gives an error.  Did anyone else notice this?


Line 22:             @Html.LabelFor(model => model.Hostnames, htmlAttributes: new { @class = "control-label col-md-2" }) Line 23:             <div class="col-md-10"> Line 24:                 @Html.ListBoxFor(model => model.Hostnames, ViewBag.HostNames as IEnumerable<SelectListItem>, new { htmlAttributes = new { @class = "form-control" } }) Line 25:                 @Html.ValidationMessageFor(model => model.Hostnames, "", new { @class = "text-danger" }) Line 26:             </div>
--
